sentry-android-core/src/main/java/io/sentry/android/core/internal/util/SentryFrameMetricsCollector.java

```diff
@@ -140,6 +145,13 @@ public SentryFrameMetricsCollector(
                   ? window.getContext().getDisplay().getRefreshRate()
                   : window.getWindowManager().getDefaultDisplay().getRefreshRate();
 
+          final long expectedFrameDuration = (long) (oneSecondInNanos / refreshRate);
+          final long totalFrameDuration = frameMetrics.getMetric(FrameMetrics.TOTAL_DURATION);
+
+          // if totalDurationNanos is smaller than expectedFrameTimeNanos,
+          // it means that the frame was drawn within it's time budget, thus 0 delay
+          final long delayNanos = Math.max(0, totalFrameDuration - expectedFrameDuration);
```

Comment:
i guess we want to use the `cpuDuration` here, as well
[JankStats's uiDuration](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:metrics/metrics-performance/src/main/java/androidx/metrics/performance/JankStatsApi24Impl.kt;l=74-82) is our cpuDuration, and they use it to calculate jank